benchmarking/lance/generate.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import annotations
+
+import os
+
+import daft
+from daft.io import IOConfig, S3Config
+
+DATASET_PATH = os.getenv("DATASET_PATH", "s3://xx/ai/dataset/ILSVRC2012/")
+LANCE_PATH = os.getenv("LANCE_PATH", "s3://xx/ai/dataset/ILSVRC2012/")
+AWS_ENDPOINT = os.getenv("AWS_ENDPOINT", "https://xx.xx")
+AWS_KEY_ID = os.getenv("AWS_KEY_ID", "xx")
+AWS_ACCESS_KEY = os.getenv("AWS_ACCESS_KEY", "xx==")
```

Comment:
can we use the standard AWS env vars here. `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`